labs/query_enriched_event_data_with_spark/README.md

```diff
@@ -0,0 +1,65 @@
+# Querying Optimizely Enriched Event data with Apache Spark
+
+In the [Enriched Events](../enriched_events) Lab we learned how to download [Enriched Event Data](https://docs.developers.optimizely.com/web/docs/enriched-events-export).  In this Lab we'll learn how to query that data with [Apache Spark](https://spark.apache.org/).
```

Comment:
Not sure what we should do with relative links `../enriched_events`, but I doubt this will work on the resulting labs page even if there was a lab in this repo under `enriched_events`. One option is to require all links to be absolute.

Reply:
Agreed; I'll remove this reference until we've published that lab and replace it with an absolute URL when we do.

Reply:
Addressed this in ab2d23d